keps/sig-cli/0031-kustomize-integration.md

```diff
@@ -199,35 +409,50 @@ Most implementation will be in cli-runtime
 - [x] vendor `kustomize/pkg` into kubernetes
 - [x] copy `kustomize/k8sdeps` into cli-runtime
 - [x] Implement a Visitor for kustomization directory which
-   - execute kustomize build to get a list of resources
-   - write the output to a StreamVisitor
+ - execute kustomize build to get a list of resources
+ - write the output to a StreamVisitor
 - [x] When parsing filename parameters in FilenameParam, look for kustomization directories
 - [ ] documentation:
-  - update the examples in kubectl commands
-  - Improve help messages or documentations to list kubectl subcommands that can work with kustomization directories
+ - update the examples in kubectl commands
+ - Improve help messages or documentations to list kubectl subcommands that can work with kustomization directories
 
 ## Alternatives
```

Comment:
This is now mixed up, alternatives should be proposed next to actual proposals, or we move those proposals that were not picked over here. Now, we have 2 places where alternatives are being discussed, which makes this document hard to follow.